database/main.py

```diff
@@ -0,0 +1,31 @@
+# main.py
+
+import sqlalchemy as sa
+
+from fastapi import FastAPI
+from sqlalchemy.orm import sessionmaker
+
+from config import DATABASE_URL
+from release_dao import readRelease, getkeys
+

```

Comment:
Please import the `logging` module and let's improve the debuggability of our system with some helpful logging lines:
```
import os
import logging

logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
logging.getLogger('sqlalchemy.engine').setLevel(logging.INFO)
log = logging.getLogger(__name__)
```